src/Eamlight.js

```diff
@@ -68,10 +68,16 @@ class Eamlight extends Component {
 
         const eqpRegex = ["/asset", "/position", "/system", "/location"].map(e => `${e}/:code(.+)?`)
 
+        const selectedTheme = ({
```

Comment:
See https://github.com/cern-eam/eam-components/pull/48 review.

We could avoid doing this association here.